client/src/components/App.jsx

```diff
@@ -7,12 +12,44 @@ class App extends React.Component {
     this.state = {
       photos: [],
     };
+
+    this.getPhotos = this.getPhotos.bind(this);
+  }
+
+  // need to create get to update state with data
+  componentDidMount() {
+    this.getPhotos();
+  }
+
+  getPhotos() {
+    return axios.get('/api/photos/10')
```

Comment:
what's the purpose of returning here?  You've already set state

Reply:
I had to do this because of the tests. My tests wouldn't pass for some reason they are when I do this. I found this solution online, but I am not all the way sure yet about the reason why this works the way it does.